### PR TITLE
test: remove use of `fs` in tests

### PR DESCRIPTION
**Pull request by mvadari**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: packages/xrpl/test/webpack.config.js [hunk 1]
```diff
@@ -100,15 +100,11 @@ function webpackForTest(testFileName) {
       extensions: ['.ts', '.js', '.json'],
       fallback: {
         module: false,
-        assert: require.resolve('assert-browserify'),
```
Comment (ckniffen) [inline packages/xrpl/test/webpack.config.js:103]: I think assert should still be required.  Maybe we dont have a test that hits any assets.

**Review by ckniffen — APPROVED** [review]

File: packages/xrpl/test/webpack.config.js (hunk 1 above)
Comment (mvadari) [reply packages/xrpl/test/webpack.config.js:103]: There's another assert webpack config a couple of lines below - we don't need two

**Comment by mvadari** [comment]
Karma depends on `fs`, so this won't work.

**Comment by mvadari** [comment]
Karma depends on `fs`, so this won't work.

**Comment by ckniffen** [comment]
> Karma depends on `fs`, so this won't work.

Does it still need the browserify version?

**Comment by mvadari** [comment]
> > Karma depends on `fs`, so this won't work.
> 
> Does it still need the browserify version?

Yes, apparently (tests were still failing on my last commit). Feel free to reopen this and take it over if you find a solution.

**Pull request closed without merging**